process-app/src/main/kotlin/com/alfresco/content/process/FormViewModel.kt

```diff
@@ -91,7 +91,7 @@ class FormViewModel(
         }
     }
 
-    fun updateFieldValue(fieldId: String, newValue: String, state: FormViewState) {
+    fun updateFieldValue(fieldId: String, newValue: Any?, state: FormViewState) {
```

Comment:
Why did you make it any?

Reply:
It's common function and accept any time of value